Automattic/newspack-plugin
# fix: correct thank you message text domain

### All Submissions:


### Changes proposed in this Pull Request:

This PR corrects the text domain in a 'thank you' donation string, and adds a long-overdue update to the plugin's POT file. 

See 1200550061930446-as-1204849686104010

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Confirm that the text domain for the "Thank you for your donation!" string is now for this plugin, and not the blocks.
3. Confirm the POT file is up to date (the plugin doesn't have any translations bundled with it, so there isn't much to test!). 

### Other information:
